dockerfile

```diff
@@ -172,25 +172,21 @@ RUN if [ -n "${SENTRY_AUTH_TOKEN}" ] && [ -n "${SENTRY_ORG}" ] && [ -n "${SENTRY
 
 # Build common package
 FROM prod-builder AS build-common
-COPY packages/common/tsconfig.json ./packages/common/
-COPY packages/common/tsconfig.build.json ./packages/common/
-COPY packages/common/tsconfig.esm.json ./packages/common/
-COPY packages/common/tsconfig.cjs.json ./packages/common/
-COPY packages/common/tsconfig.types.json ./packages/common/
+COPY packages/common/tsconfig*.json ./packages/common/
 COPY packages/common/src/ ./packages/common/src/
 RUN pnpm -F @lightdash/common build
 
 # Build warehouses package
 FROM prod-builder AS build-warehouses
-COPY --from=build-common /usr/app/packages/common/dist/ ./packages/common/dist/
+COPY --from=build-common /usr/app/packages/common/ ./packages/common/
```

Comment:
need to copy src code as well 🤷 I'm sure we can adjust tsconfigs to not need this but just want to fix the build right now